examples/decoding/plot_decoding_rsa.py

```diff
@@ -0,0 +1,143 @@
+""" Representational Similarity Analysis
+======================================
+
+Representational Similarity Analysis is used to perform summary statistics
+on supervised classifications where the number of classes is relatively high.
+It consists in characterizing the structure of the confusion matrix to infer
+the similarity between brain responses and serve as a proxy for characterizing
+the space of mental representations [1-3].
+
+In this example, we perform RSA on spiking data from [4] where the macaque was
+presented to seven classes of pictures located in one of three spatial
+positions.
+
+References
+----------
+[1] Shepard, R. "Multidimensional scaling, tree-fitting, and clustering."
+ Science 210.4468 (1980): 390-398.
+[2] Laakso, A. & Cottrell, G.. "Content and cluster analysis:
+ assessing representational similarity in neural systems." Philosophical
+ psychology 13.1 (2000): 47-76.
+[3] Kriegeskorte, N., Marieke, M., & Bandettini.  P. "Representational
+ similarity analysis-connecting the branches of systems neuroscience."
+ Frontiers in systems neuroscience 2 (2008): 4.
+[4] Cichy, R. M., Pantazis, D., & Oliva, A. "Resolving human object recognition
+in space and time. Nature neuroscience" (2014): 17(3), 455-462
+"""
+# Authors: Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+
+print(__doc__)
+
+import os.path as op
+import numpy as np
+from pandas import read_csv
+import matplotlib.pyplot as plt
+
+from mne import find_events, Epochs, pick_types
+from mne.io import Raw
+
+from sklearn.model_selection import StratifiedKFold
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import roc_auc_score
+from sklearn.manifold import MDS
+
+data_path = '/media/DATA/Pro/Projects/NewYork/cichy/'
+###############################################################################
+
+# Define stimulus - trigger mapping
+fname = op.join(data_path, 'visual_stimuli.csv')
+conds = read_csv(fname)
+triggers = set(conds['trigger'])
+inv_conds = dict()
+for val in triggers:
+    sub = conds.query('trigger == %i' % val)
+    inv_conds[val] = sub['condition'].iloc[0]
+
+# Read MEG data
+fname = op.join(data_path, 'sample_subject_0_tsss_mc.fif')
+raw = Raw(fname)
```

Comment:
Raw only reads fif files. Now we can read many files and I've seen people trying Raw on edf files... We use now everywhere (normally) read_raw_fiff